### PR TITLE
[Wallet] Stop screen jumping for Wallet home screen and Verify Education screen

### LINKED ISSUE #696: Wallet home screen jumps when connecting banner hides
### Expected Behavior

No jumps
This would be achieved by keeping that whitespace there or having it placed over the navigation header. I prefer the latter. 

### Current Behavior

Banner is shown when app starts, then the content jumps upwards when it hides

### PR DESCRIPTION
**Pull request by sallyjyl**
### Description
Move the disconnect banner up to the navigation header for wallet home and verify education screen such that there is no screen jumping when the banner hides.

### Tested
new
new2
verifyPhone
verifyPhone2

### Related issues

- Fixes #696 

### Backwards compatibility
Yes

### CODE REVIEW

File: packages/mobile/src/navigator/Headers.tsx [hunk 1]
```diff
@@ -12,23 +12,24 @@ export const nuxNavigationOptions = {
   },
   headerLeftContainerStyle: { paddingHorizontal: 20 },
   headerLeft: <BackButton />,
-  headerRightContainerStyle: { paddingRight: 15 },
-  header: (
-    <View style={{ paddingVertical: 30 }}>
+  headerRightContainerStyle: { paddingHorizontal: 20 },
+  headerRight: <View />,
+  headerTitle: (
+    <View style={{ paddingTop: 5, alignItems: 'center', flex: 1, justifyContent: 'center' }}>
```
Comment (jmrossy) [inline packages/mobile/src/navigator/Headers.tsx]: Nit, could you set these values on `headerTitleContainerStyle` instead of creating a view wrapper here?

**Review by jmrossy — COMMENTED** [review]
Nice! One comment below

File: packages/mobile/src/navigator/Headers.tsx [hunk 2]
```diff
@@ -15,10 +15,16 @@ export const nuxNavigationOptions = {
   headerRightContainerStyle: { paddingHorizontal: 20 },
   headerRight: <View />,
   headerTitle: (
-    <View style={{ paddingTop: 5, alignItems: 'center', flex: 1, justifyContent: 'center' }}>
+    <View>
```
Comment (jmrossy) [inline packages/mobile/src/navigator/Headers.tsx]: Do we still need this view wrapper now?

Comment (sallyjyl) [reply packages/mobile/src/navigator/Headers.tsx]: nope :)

**Review by jmrossy — APPROVED** [review]
Nice!

**Pull request merged**